### PR TITLE
feat(slack-notification): debtor receives slack notification after co…

### PR DESCRIPTION
**Pull request by KyuubiNoKitsune**
el deudor recibe una notificación luego de confirmar la deuda.  Se agregó specs para el endpoint que notifica.

### CODE REVIEW

File: spec/controllers/api/v1/slack_controller_spec.rb [hunk 1]
```diff
@@ -25,5 +27,9 @@
       get :get_users
       expect(JSON.parse(response.body).keys).to contain_exactly("slack")
     end
+    it "notifies user" do
+      post :notify_user, params: { user_name: slack_user.profile["display_name_normalized"], user_id: slack_user.profile["id"] }
```
Comment (monkeyci) [inline spec/controllers/api/v1/slack_controller_spec.rb]: Line is too long. [128/100]

File: app/javascript/api/invoices.js [hunk 2]
```diff
@@ -56,4 +56,13 @@ export default {
       url: path,
     });
   },
+  notifyUser(user_name, user_id) {
+    const path = 'api/v1/slack';
+
+    return api({
+      method: 'post',
+      url: path,
+      data: {user_name, user_id},
```
Comment (monkeyci) [inline app/javascript/api/invoices.js]: A space is required after '{'                   object-curly-spacing<br>Identifier 'user_name' is not in camel case     camelcase<br>Identifier 'user_id' is not in camel case       camelcase<br>A space is required before '}'                  object-curly-spacing

File: app/javascript/api/invoices.js [hunk 3]
```diff
@@ -56,4 +56,13 @@ export default {
       url: path,
     });
   },
+  notifyUser(user_name, user_id) {
```
Comment (monkeyci) [inline app/javascript/api/invoices.js]: Identifier 'user_name' is not in camel case     camelcase<br>Identifier 'user_id' is not in camel case       camelcase

File: app/controllers/api/v1/slack_controller.rb [hunk 4]
```diff
@@ -10,14 +10,23 @@ def get_users
         member.profile.display_name_normalized != "todobot" &&
         member.profile.display_name_normalized != "trello"
     end
-    user_names = members.map { |m| m.profile.display_name_normalized }
+    user_names = members.map { |m| { "display_name_normalized": m.profile.display_name_normalized, "id": m.id } }
+    puts user_names
```
Comment (monkeyci) [inline app/controllers/api/v1/slack_controller.rb]: Do not write to stdout. Use Rails's logger if you want to log.

File: app/controllers/api/v1/slack_controller.rb [hunk 5]
```diff
@@ -10,14 +10,23 @@ def get_users
         member.profile.display_name_normalized != "todobot" &&
         member.profile.display_name_normalized != "trello"
     end
-    user_names = members.map { |m| m.profile.display_name_normalized }
+    user_names = members.map { |m| { "display_name_normalized": m.profile.display_name_normalized, "id": m.id } }
```
Comment (monkeyci) [inline app/controllers/api/v1/slack_controller.rb]: Line is too long. [113/100]

File: app/controllers/api/v1/slack_controller.rb [hunk 6]
```diff
@@ -10,14 +10,24 @@ def get_users
         member.profile.display_name_normalized != "todobot" &&
         member.profile.display_name_normalized != "trello"
     end
-    user_names = members.map { |m| m.profile.display_name_normalized }
+    user_names = members.map { |m|
```
Comment (monkeyci) [inline app/controllers/api/v1/slack_controller.rb]: Avoid using {...} for multi-line blocks.

File: app/javascript/components/app-invoice.vue [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="invoice">
-    <errorNotification v-if="!loading && invoice.paymentRequest" />
-    <!--  <errorNotification /> -->
+    <!--  <errorNotification v-if="!loading && invoice.paymentRequest" />-->
```
Comment (rjherrera) [inline app/javascript/components/app-invoice.vue]: cual de los dos es el que corresponde? trataría de solo dejar el que va a ir y el comentario eliminarlo

**Comment by KyuubiNoKitsune** [comment]
[Al confirmar fiada, que se notifique por Slack](https://trello.com/c/uQvD8IBj/145-al-confirmar-fiada-que-se-notifique-por-slack)

**Pull request merged**